src/sas/qtgui/Utilities/GuiUtils.py

```diff
@@ -1369,15 +1369,19 @@ def readProjectFromSVS(filepath):
     loader = Loader()
     loader.associate_file_reader('.svs', Reader)
     temp = loader.load(filepath)
-    state_reader = Reader()
-    data_svs, state_svs = state_reader.read(filepath)
 
-    output = []
+    # CRUFT: SasView 4.x uses a callback interface to register bits of state
+    state_svs = []
+    def collector(state=None, datainfo=None, format=None):
+        if state is not None:
+            state_svs.append(state)
+    state_reader = Reader(callback=collector)
```

Comment:
This currently breaks 5.0
`ERROR: Error while reading the project file: __init__() got an unexpected keyword argument 'callback'`